op-e2e/actions/l2_challenger_test.go

```diff
@@ -76,28 +133,53 @@ interaction:
 	require.NoError(rt.t, err)
 	require.Equal(rt.t, chal.StatusNone, status)
 
-	// check bond amount doubled after challenge proven
-	bond, err := rt.valPoolContract.GetBond(nil, rt.outputIndex)
-	require.NoError(rt.t, err)
-	require.Equal(rt.t, big.NewInt(2*rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt().Int64()), bond.Amount)
+	if version == defaultValidatorSystemVersion {
+		// check bond amount doubled after challenge proven
+		bond, err := rt.valPoolContract.GetBond(nil, rt.outputIndex)
+		require.NoError(rt.t, err)
+		require.Equal(rt.t, big.NewInt(2*rt.dp.DeployConfig.ValidatorPoolRequiredBondAmount.ToInt().Int64()), bond.Amount)
+	} else if version == defaultValidatorSystemVersion+1 {
```

Comment:
How about making validator system v2 const as well? It would be nice to have two versions of `validatorV1` and `validatorV2` to compare, rather than `defaultValidatorSystemVersion`.